browser/src/ConstraintTable/GnomadNonCodingConstraintTableVariant.tsx

```diff
@@ -0,0 +1,126 @@
+  const surroundingLocation = `${chrom}-${variantLocation - 20000}-${variantLocation + 20000}`
+
+  return (
+    <>
+      {/* TODO:(rgrant) link to blog post here, when this is ready */}
+      {/* <p>
+        Read more about this <Link to={'/'}>here</Link>
+      </p> */}
+      <div>
+        {`Genomic constraint values displayed are for the region: ${chrom}-${nonCodingConstraint.start}-${nonCodingConstraint.stop}`}
+      </div>
+      <Table>
+        <thead>
+          <tr>
+            <th scope="col">
+              {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+              <TooltipAnchor tooltip="Expected variant counts is calculated using a context-dependent mutability for each window by summing all trinucleotide mutabilities within the 1kb sequence. In addition, 17 genomic features shown to influence mutation rate, such as replication timing and recombination rate, were used alongside the local sequence context to predict the expected value.">
+                {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+                <TooltipHint>Expected</TooltipHint>
+              </TooltipAnchor>
+            </th>
+            <th scope="col">
+              {/* @ts-expect-error TS(2322) FIXME: Type '{ children: Element; tooltip: string; }' is ... Remove this comment to see the full error message */}
+              <TooltipAnchor tooltip="Observed variants is the number of rare (MAF<=1%) varaints in this 1kb window as identified in gnomAD v3.1.2">
+                {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+                <TooltipHint>Observed</TooltipHint>
+              </TooltipAnchor>
+            </th>
+            <th scope="col">Constraint</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>{renderRoundedNumber(nonCodingConstraint.expected)}</td>
+            <td>{nonCodingConstraint.observed}</td>
+            <td>
+              Z ={' '}
+              {renderRoundedNumber(nonCodingConstraint.z, {
+                precision: 2,
+                tooltipPrecision: 3,
+                highlightColor: null,
+              })}
+              <br />
+              o/e ={' '}
+              {renderRoundedNumber(nonCodingConstraint.oe, {
+                precision: 2,
+                tooltipPrecision: 3,
+                highlightColor: null,
+              })}
+            </td>
+          </tr>
+        </tbody>
+      </Table>
+      <ViewSurroundingRegion>
+        <p>View the genomic constraint values for the 40kb region surrounding this variant</p>
+        <p>
+          <Link
+            to={`/region/${surroundingLocation}?variant=${variantId}&dataset=gnomad_r3`}
```

Comment:
In short, yes - with a caveat to come back to refactor `<Link>`'s code. Not ideal, I know.

This is a quirk of the custom `<Link>` wrapper not working (I believe) as intended. We don't often use multiple query strings, and the default behavior of the `<Link>` is to preserve the query string of the dataset, when navigating around pages. When re-adding the query string to the new link, it adds `?dataset=...` rather than `&dataset=...`, causing query string to incorrectly drop the intended information of the dataset.

To quickly workaround this, this link is flagged to not `preserveSelectedDataset`, and a fixed dataset was added.